foam_controlwrapper/tests/test_foam_interface.py

```diff
@@ -0,0 +1,292 @@
+""" test foam native interface
```

Comment:
Since these are tests for the foam_interface package they should be moved to the foam_interface package